src/java.scripting/share/classes/module-info.java

```diff
@@ -30,6 +30,8 @@
  * command-line script shell, <em>{@index jrunscript jrunscript tool}</em>,
  * that supports executing JavaScript and other languages if its corresponding
  * script engine is installed.
+ * <p> The {@code jrunscript} tool is deprecated for removal since JDK 24
```

Comment:
I think "since JDK 24" can be dropped as not needed and consistent with other deprecation message.

Reply:
Hello Mandy, I've now updated the PR to remove that part.